src/Tests/Framework/ManagedElasticsearch/NodeSeeders/MachineLearningSeeder.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.IO;
+using Nest;
+using Tests.Framework.ManagedElasticsearch.Nodes;
+using Tests.Framework.MockData;
+
+namespace Tests.Framework.ManagedElasticsearch.NodeSeeders
+{
+	public class MachineLearningSeeder
```

Comment:
This appears to be unused, intentionally? Can be removed?

Reply:
It is used, fixed.